packages/core/test/http-client/http-client.spec.ts

```diff
@@ -67,8 +67,7 @@ describe('generic http client', () => {
         headers: {
           authorization: 'Basic VVNFUk5BTUU6UEFTU1dPUkQ=',
           'sap-client': '001',
-          'Proxy-Authorization': 'jajaja',
-          'SAP-Connectivity-Authentication': 'yeahyeah'
+          'Proxy-Authorization': 'jajaja'
```

Comment:
perhaps we should use more mature dummy values or more funny ones like `Testination` or `BernieAndCert`. `jajaja` is a bit weak.

Reply:
wasn't me 🤓but I also thought of this...